samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java

```diff
@@ -30,14 +34,16 @@
  * </p>
  */
 public class SuggestedActionsBot extends ActivityHandler {
-    public static final String WELCOMETEXT =
-        "This bot will introduce you to suggestedActions." + " Please answer the question:";
+    public static final String WELCOME_TEXT =
+        "This bot will introduce you to suggestedActions. Please answer the question:";
 
     @Override
     protected CompletableFuture<Void> onMembersAdded(
         List<ChannelAccount> membersAdded,
         TurnContext turnContext
     ) {
+        // Send a welcome message to the user and tell them what actions 
+        // they may perform to use this bot
         return sendWelcomeMessage(turnContext);
```

Comment:
Just to confirm, this is correctly accessing to the `static` method? I mean, this method is not `static`, should it be `SuggestedActionsBot.sendWelcomeMessage(turnContext);`?

Reply:
I'll follow the [multilingual sample](https://github.com/southworks/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java#L75) and call it through the class